src/nlr/LPFormulator.cpp

```diff
@@ -279,10 +279,19 @@ void LPFormulator::optimizeBoundsWithLpRelaxation( const Map<unsigned, Layer *>
 
     gurobiStart = TimeUtils::sampleMicro();
 
+    bool skipTightenLb = false; // If true, skip lower bound tightening
+    bool skipTightenUb = false; // If true, skip upper bound tightening
+
+    unsigned skipTighterLowerBoundCounter = 0;
```

Comment:
Since these two variables are used only for printing, maybe we should discard them.

Reply:
I changed logging to print not counter but which neuron is skipped.